packages/stateful/proposal-module-adapter/adapters/DaoProposalSingle/components/ProposalStatusAndInfo.tsx

```diff
@@ -212,6 +212,9 @@ const InnerProposalStatusAndInfo = ({
       : []),
   ]
 
+  const proposalCompleted =
+    proposal.status !== Status.Open && proposal.status !== Status.Passed
```

Comment:
this means that a proposal is only completed (and thus disallows votes cast) when it is rejected, executed(/failed execution), or closed. looking at the contract code, it seems like even passed proposals can be voted on before expiry. am I misunderstanding that, or is there a reason you are preventing passed proposals from being voted on here?

i would think the condition here should be `NOT open AND expired`.

also nit: `proposalCompleted` here is a bit vague as there's additional context relevant here; an explanatory comment might help, or renaming to something like `proposalOpenForVoting`

Reply:
ah you're right. thanks!